scapy/contrib/bgp.uts

```diff
@@ -297,7 +297,7 @@ str(BGPOptParam()) == b'\x02\x00'
 
 = BGPOptParam - Instantiation with specific values (1)
 str(BGPOptParam(param_type = 1)) == b'\x01\x00'
-str(BGPOptParam(param_type = 1, param_value = BGPAuthenticationInformation())) == b'\x01\x04'
+str(BGPOptParam(param_type = 1, param_value = BGPAuthenticationInformation())) == b'\x01\x00'
```

Comment:
Explanation: BGP was using i2m method that did not check if the field was empty. Here: "\x04" stands for `len(...) = 4` because the string "None" was passed to the function. It's a bug fixed indirectly